randovania/games/samus_returns/gui/ui_files/msr_game_export_dialog.ui

```diff
@@ -339,20 +308,58 @@
      </property>
     </spacer>
    </item>
+   <item row="12" column="0">
+    <widget class="QCheckBox" name="auto_save_spoiler_check">
+     <property name="text">
+      <string>Include a spoiler log on same directory</string>
+     </property>
+    </widget>
+   </item>
+   <item row="3" column="0" colspan="2">
+    <widget class="QLabel" name="input_exheader_label">
+     <property name="text">
+      <string>Input exheader.bin (Only required for automatic item tracker and multiworld)</string>
```

Comment:
this should link to the website guied

Reply:
That's a good idea. For consistency it should be added to Dread RomFS, MSR RomFS, *insert game which is not "place iso" and done" here*.

Reply:
AFAIK dread romfs and msr romfs already link the guide.
At the very least dread romfs does because I added a PR for it.